src/index.html

```diff
@@ -7,6 +7,130 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar-start-day--mon calendar-month-length--31">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--mon calendar--month-length--31">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/570bec87-cb2a-4a2f-8a79-143d5ed10d66)